app/src/screens/About/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Linking,
+  TouchableHighlight,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#F4F4F6',
+    height: '100%',
+    display: 'flex',
+    alignContent: 'center',
+    marginTop: 100,
+    marginHorizontal: 40,
+  },
+  header: {
+    color: '#121C2D',
+    fontSize: 24,
+    fontWeight: '700',
+    marginBottom: 10,
+  },
+  section: {
+    marginBottom: 50,
+  },
+  highlight: {
+    color: '#0263E0',
+  },
+  channel: {
+    color: '#0263E0',
+    textDecorationLine: 'underline',
+  },
+  issueDescription: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+});
+
+const About: React.FC = () => {
+  const VOICE_BLOCKS_URL = 'https://twilio.slack.com/archives/C020DUH6R1B';
+  const VOICE_BLOCKS_CHANNEL = '#eng-voice-blocks';
```

Comment:
Can we make these environment variables? That way we (or customers) don't need to change the code later on.

Reply:
Also, name them `SLACK_URL` and `SLACK_CHANNEL_NAME`